test_schemas/en/test_textfield_suggestions.json

```diff
@@ -36,15 +36,38 @@
                                         "id": "country-answer",
                                         "label": "What is your country of birth?",
                                         "mandatory": false,
-                                        "suggestions_url": "https://cdn.eq.census-gcp.onsdigital.uk/data/v5.0.0/gb/en/countries-of-birth.json",
+                                        "suggestions": {
+                                            "url": "https://cdn.eq.census-gcp.onsdigital.uk/eq-lookup-suggestions-data/v5.3.0/gb/en/countries-of-birth.json",
+                                            "allow_multiple": true
```

Comment:
Should this be `false`?

Reply:
Yes, sorry, only second question is tested for multiple lookups.